homeassistant/components/zwave/websocket_api.py

```diff
@@ -0,0 +1,32 @@
+"""Web socket API for Z-Wave."""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+
+from .const import DATA_NETWORK
+
+_LOGGER = logging.getLogger(__name__)
+
+TYPE = 'type'
+ID = 'id'
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command({
+    vol.Required(TYPE): 'zwave/network_status'
+})
+def websocket_network_status(hass, connection, msg):
+    """Get Z-Wave network status."""
+    network = hass.data[DATA_NETWORK]
+    connection.send_result(msg[ID], {
+        'state': network.state,
```

Comment:
What's this value and why would we need both?

Reply:
`state` is a numeric value for the state of the network provided by OZW (basically if state >= 7 the network is running), then `state_str` is a text description of the status.

Reply:
So if we want to support localization, you should only return the network state and add translation strings for all the different states in the frontend.